learn/why-ballerina/reliable-maintainable.md

```diff
@@ -57,11 +57,11 @@ remote function write(byte[] bytes) returns error? {}
 
 ### Error Checking 
 
-Ballerina has language constructs for explicit error checking. Both explicit error reporting and checking improves code readability and maintainability simply because they are explicit. When you read Ballerina code, you can quickly notice code that reports errors and checks for errors.  
+Ballerina has language constructs for explicit error checking. Both explicit error reporting and checking improve code readability and maintainability simply because they are explicit. When you read Ballerina code, you can quickly notice the code, which reports errors and you can check for errors.  
```

Comment:
```suggestion
Ballerina has language constructs for explicit error checking. Both explicit error reporting and checking improve code readability and maintainability simply because they are explicit. When you read Ballerina code, you can quickly notice the code, which reports errors, and you can check for errors.  
```